bennu-portal/src/main/java/org/fenixedu/bennu/portal/domain/MenuItem.java

```diff
@@ -334,4 +343,28 @@ protected User getBennuUser(ApplicationUser appUser) {
         return appUser != null ? User.findByUsername(appUser.getUsername()) : null;
     }
 
+    @Override
+    public Layout getLayoutObject() {
+        return Optional.ofNullable(layoutProviderRegistrySupplier.get())
+                .map(layoutProviderRegistry -> layoutProviderRegistry.getLayoutProviders().values().iterator())
```

Comment:
@francisco-esteves also I think it would be better for us to have a comment just saying that in Fenix currently only one layout provider exists and we can do the next() without worrying that much about it, that means we could even drop the filter since we know it will have one and only one